src/cli/cli.cc

```diff
@@ -1244,6 +1254,7 @@ int main (const int argc, const char* argv[]) {
         split(settings["android_native_sources"], ' ')
       ) {
         auto filename = fs::path(file).filename();
+        log(String("Android NDK source: " + String(targetPath / file)).c_str());
```

Comment:
Can you remove this? We actually removed it in another commit

Reply:
👍